controllers/rabbitmqcluster_controller_test.go

```diff
@@ -1206,6 +1206,21 @@ var _ = Describe("RabbitmqclusterController", func() {
 
 			Expect(sts.Spec.Template.Spec.HostNetwork).To(BeFalse())
 			Expect(sts.Spec.Template.Spec.Volumes).To(ConsistOf(
+				corev1.Volume{
+					Name: "rabbitmq-admin",
```

Comment:
Unless this user is always supposed to exist and always have administrative privileges, I'd suggest naming this "rabbitmq-default-user".

Reply:
`rabbitmq` with password `rabbitmq` sounds most reasonable to me. I never really understood the reasoning behind `guest`. Do you know where that name came from @michaelklishin?

Reply:
This user will be admin by default as per the default config for RabbitMQ of `default_user_tags.administrator = true`. As for whether it should be admin by design for the operator, I'd say it's the default user first, that happens to be admin by default unless otherwise configured.

Reply:
Oh, I now understand what the value really is.

From the protocol committee back in the day. "guest" is a reasonable name and is so widely known and documented in the RabbitMQ community that I'd keep it.

So yeah, I'd go with `guest`/`guest` or `administrator`/`administrator` if we intend this user to be the "root" for the cluster.

Reply:
If this is the default user, which is there for convenience and testability, I'd say we can go with `guest`/`guest` as widely known credentials specifically for RabbitMQ. The risk is, well, as with all well known credentials :/

Reply:
Currently in the operator, we override `guest`/`guest` with randomly generated strings. They're stored in a K8s Secret suffixed with `-admin`.

Reply:
That sounds good.

Reply:
You raise a valid point though. The only reason this user is an admin is by the default value of `default_user_tags.administrator`. There's no cluster-specific reason you'd need this user to be an admin; while you may need an admin to create resources or other users on the cluster, you could also just as easily make the default user not an admin through configuration. If you did this, the Status field on the CR would report the 'Admin Credentials' for a user that isn't an admin, and the secret containing the now-non-admin user would be called `XXXXXX-admin`.

I'll raise an issue to switch to referring it as `default_user`, and maybe displaying its permissions in the CR Status field, and we can discuss whether we want to play the story.